app/helpers/admins_helper.rb

```diff
@@ -10,4 +10,9 @@ def channel_url(channel)
       raise "Unsupported channel type"
     end
   end
+
+  def note_creator(note)
```

Comment:
I'd prefer to keep the admin helper skinnier and the caller more transparent, in other words, I'd prefer to remove this.